src/components/MessagesList/MessagesList.vue

```diff
@@ -28,42 +28,28 @@ the DynamicScroller component, whose docs you can find [here.](https://github.co
 </docs>
 
 <template>
-	<DynamicScroller
-		:items="messagesList"
-		:min-item-size="60"
+	<virtual-list :size="40" :remain="8" :variable="true"
```

Comment:
Why those specific numbers?

Reply:
Those are defaults and probably will need to be tweaked in the future, they represent how many items are loaded outside the view-port and their initial height.

Reply:
Shouldn't initial height be like one message? is it 40?

Reply:
Maybe add a comment to explain what those are :)

Reply:
if this is px, then the min height is wrapper padding + font size, no?
Should be something around 25px?

Reply:
But why would you use the min height?

Reply:
I don't know, you mentionned that it was supposed to be their initial height :p

So since you don't know the exact height, the initial we know is the min?
If it's working, then all good :)
I just did not understood those values :see_no_evil:

Reply:
This is it :)
![Screenshot_20191018_101501](https://user-images.githubusercontent.com/26852655/67077768-321f2f80-f190-11e9-8f20-8ad12f6a8483.png)